src/views/product-tutorials-view/server.ts

```diff
@@ -163,24 +145,23 @@ export async function getProductTutorialsViewProps(
 	 * here.
 	 */
 	const { description, docsUrl, id, name, slug } = product
+
+	/**
+	 * Return static props
+	 */
 	return {
-		props: stripUndefinedProperties<$TSFixMe>({
-			metadata: {
-				title: 'Tutorials',
-			},
+		props: stripUndefinedProperties({
 			data: {
 				pageData,
-				allCollections: sitemapCollections,
-				inlineCollections,
-				inlineTutorials,
+				sitemapCollections,
 			},
 			layoutProps,
 			product: {
 				...productData,
 				description,
 				docsUrl,
 				id,
-				name,
+				name: name as LearnProductName,
```

Comment:
Is this similar to the 'hcp' issue above?

Reply:
Yep, exactly!